src/app/ngx-slimscroll/directives/slimscroll.directive.ts

```diff
@@ -71,6 +72,13 @@ export class SlimScrollDirective implements OnChanges, OnDestroy {
     this.mutationThrottleTimeout = 50;
   }
 
+  ngOnInit() {
+    // setup if no changes for enabled for the first time
```

Comment:
Take an example, user didn't touch the `enabled` input.
```
<div slimscroll options="[options]"></div>
```
then there will be no changes for `enabled`, and no setup for the scrollbar.
we have to setup for `enabled` with default `true` on init. Of course we could merge that logic into `ngOnChanges` method, but it would become harder to read with complex condition, IMO.
So i make it into `ngOnInit`. That's a bug found by my colleague.

Reply:
ngOnChanges is fired before ngOnInit. That's a prerequisite

Reply:
cool, thanks!